docs/dev-guide/integration/rest-api/repo/cud.rst

```diff
@@ -233,6 +233,12 @@ dictate how the distributor will function when it publishes content. The possibl
 configuration values are contingent on the type of distributor being added; each
 distributor type will support a different set of values relevant to how it functions.
 
+.. note::
+        There is a new optional parameter in the distributor configuration that is usable
```

Comment:
s/a new/an/
s/parameter in the distributor configuration/distributor configuration parameter/